packages/App/src/onBoardingPages/OnBoardingItem.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react-native/no-inline-styles */
+import React, {FC} from 'react';
+import {View, Image, useWindowDimensions, StyleSheet, Text} from 'react-native';
+import {OnBoardingInfo} from './OnBoarding';
+
+interface Props {
+  item: OnBoardingInfo;
+}
+
+const OnBoardingItem: FC<Props> = ({item}) => {
+  const {width} = useWindowDimensions();
+
+  return (
+    <View style={[styles.container, {width}]}>
+      <Image
+        source={item.image}
+        style={[styles.img, {width, resizeMode: 'contain'}]}
```

Comment:
To prevent unnecessary re-renders you can use `useMemo` for styles computed inside of the component